doc/guides/modules.rst

```diff
@@ -10,14 +10,342 @@ system those are called *modules*. These modules must be integrated with the
+
+Allowed practices
+---------------------
+
+The following practices are allowed and should be followed in all modules.
+
+**Updating modules with a diff from upstream:**
+Upstream changes brought as a single commit (manual diff) in a PR against
+the module's main branch. This approach is simple and should be applicable
+to all modules with the downside of supressing the upstream history in the
+module repository.
+
+The commit message is expected to identify the upstream repository, the
+version to which the module is updated (upstream version, tag, commit,
+etc.), and the reason for the doing the update.
+
+**Cherry picking upstream commits:**
+Upstream changes brought in by cherry-picking the upstream commits and
+submitting a PR with these commits against the module main branch. This
+approach is applicable to modules with an upstream project Git repository.
+The main advantages of this approach are the following
+
+* the upstream repository history may be preserved in the module repository,
+  if the cherry picked commits have a pointer to the original upstream commit
+* downstream module patches are less likely to be unintentionally overwritten
+  by the synchronization process.
+
+
+Contributing to Zephyr modules
+******************************
+
+.. _modules_contributing:
+
+
+Individual Roles & Responsibilities
+===================================
+
+To facilitate management of Zephyr module repositories, the following
+individual roles are defined.
+
+**Administration:** Each Zephyr module shall have an administrator
+who is responsible for managing access to the module repository,
+for example, for adding individuals as Collaborators in the repository
+at the request of the module code owner. Module administrators are
+members of the Administrators' team, that is a group of project
+members with admin rights to Module Github repositories.
+
+**Code ownership:** Each module shall have a code owner. Module code
+owners will have the overall responsibility of the contents of a
+Zephyr module repository. In particular, a module code owner will
+
+* coordinate code reviewing in the module repository
+* be the default assignee in pull-requests against the repository's
+  main branch
+* request additional collaborators to be added to the repository, as
+  they see fit
+* regularly synchronize the module repository with its upstream
+  counterpart following the policies described in
+  :ref:`modules_synchronization`
+* be aware of security vulnerability issues in the external project
+  and update the module repository to include security fixes, as
+  soon as the fixes are available in the upstream code base
+* list any known security vulnerability issues, present in the
+  module code base, in Zephyr release notes.
+
+
+  .. note::
+
+     Module code owners are not required to be Zephyr
+     :ref:`Maintainers <project_roles>`.
+
+**Merging:** The Zephyr Release Engineering team has the right and the
```

Comment:
```suggestion
**Merger:** The Zephyr Release Engineering team has the right and the
```

Similar to "Administrator", "merging" is a responsibility, not a role.

Reply:
done